configs/logging_config.yaml

```diff
@@ -0,0 +1,43 @@
+version: 1
+formatters:
+  simple:
+    format: '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+  colored:
+    (): colorlog.ColoredFormatter
+    format: '%(bg_black)s%(log_color)s%(levelname)-8s%(reset)s %(bg_blue)s[%(pathname)s : lineno %(lineno)s]%(reset)s %(bg_cyan)s %(asctime)s %(reset)s %(bg_black)s%(log_color)s%(message)s%(reset)s'
+    datefmt: '%m-%d %H:%M:%S'
+    log_colors:
+      DEBUG: green
+      INFO: white
+      WARNING: yellow
+      ERROR: red,
```

Comment:
Super minor, but is this comma needed here? I don't see commas for other `log_colors`